stl/inc/utility

```diff
@@ -132,6 +132,8 @@ _EXPORT_STD struct piecewise_construct_t { // tag type for pair tuple arguments
     explicit piecewise_construct_t() = default;
 };
 
+_EXPORT_STD _INLINE_VAR constexpr _Ignore ignore{};
+
 _EXPORT_STD _INLINE_VAR constexpr piecewise_construct_t piecewise_construct{};

```

Comment:
This is putting `ignore` between the definitions of `piecewise_construct_t` and `piecewise_construct`. IMO `piecewise_construct_t` and `piecewise_construct` should be kept together, and `ignore` should be either before or after them.

Reply:
Understood I'll do just that